x-pack/platform/plugins/shared/dataset_quality/public/rule_types/degraded_docs/rule_form/rule_form.tsx

```diff
@@ -0,0 +1,327 @@
+      preFillProperty('comparator');
+    }
+
+    if (!ruleParams.threshold) {
+      preFillProperty('threshold');
+    }
+
+    if (!ruleParams.timeSize) {
+      preFillProperty('timeSize');
+    }
+
+    if (!ruleParams.timeUnit) {
+      preFillProperty('timeUnit');
+    }
+
+    if (!ruleParams.groupBy) {
+      preFillProperty('groupBy');
+    }
+  }, [
+    preFillProperty,
+    ruleParams.comparator,
+    ruleParams.groupBy,
+    ruleParams.name,
+    ruleParams.threshold,
+    ruleParams.timeSize,
+    ruleParams.timeUnit,
+  ]);
+
+  const onGroupByChange = useCallback(
+    (group: string | null | string[]) => {
+      const gb = group ? (isArray(group) ? group : [group]) : [];
+      setRuleParams('groupBy', gb);
+    },
+    [setRuleParams]
+  );
+
+  const derivedIndexPattern = useMemo<DataViewBase>(
+    () => ({
+      fields: dataView?.fields || [],
+      title: dataView?.getIndexPattern() || 'unknown-index',
+    }),
+    [dataView]
+  );
+
+  const onSelectDataView = useCallback(
+    (newDataView: DataView) => {
+      setDataViewError(undefined);
+      updateProperty('name', newDataView.getIndexPattern());
+      setDataView(newDataView);
+    },
+    [updateProperty]
+  );
+
+  const handleChange = useCallback(
+    (selectedOptions: Array<{ label: string }>) => {
+      const gb = selectedOptions.map((option) => option.label);
+
+      onGroupByChange([...new Set(preselectedOptions.concat(gb))]);
+    },
+    [onGroupByChange, preselectedOptions]
+  );
+
+  const getPreSelectedOptions = () => {
+    return preselectedOptions.map((field) => ({
+      label: field,
+      color: 'lightgray',
+      disabled: true,
+    }));
+  };
+
+  const getUserSelectedOptions = (group: string[] | undefined) => {
+    return (group ?? [])
+      .filter((g) => !preselectedOptions.includes(g))
+      .map((field) => ({
+        label: field,
+      }));
+  };
+
+  const selectedOptions = [...getPreSelectedOptions(), ...getUserSelectedOptions(groupBy)];
+
+  return (
+    <>
+      <EuiTitle size="xs">
+        <h5>
+          <FormattedMessage
+            id="xpack.datasetQuality.rule.alertCondition"
+            defaultMessage="Alert condition"
+          />
+        </h5>
+      </EuiTitle>
+
+      <EuiSpacer size="m" />
+
+      <DataViewSelectPopover
+        dependencies={{ dataViews, dataViewEditor }}
+        dataView={dataView}
+        metadata={{ adHocDataViewList: adHocDataViews }}
+        onSelectDataView={onSelectDataView}
+        onChangeMetaData={({ adHocDataViewList }) => {
+          onChangeMetaData({ ...metadata, adHocDataViewList });
+        }}
+      />
+      {dataViewError && (
+        <>
+          <EuiFormErrorText data-test-subj="datasetQualityRuleDataViewError">
+            {dataViewError}
+          </EuiFormErrorText>
+          <EuiSpacer size="s" />
+        </>
+      )}
+
+      <EuiExpression
+        data-test-subj="datasetQualityRuleCountExpression"
+        description={'PERCENTAGE'}
```

Comment:
is this ok being `PERCENTAGE` or should be change it to `RATE` @LucaWintergerst ?

Reply:
no percentage sounds better here